pkg/kube/client.go

```diff
@@ -695,7 +695,9 @@ func (c *client) EnvoyDo(ctx context.Context, podName, podNamespace, method, pat
 
 func (c *client) portForwardRequest(ctx context.Context, podName, podNamespace, method, path string, port int) ([]byte, error) {
 	formatError := func(err error) error {
-		return fmt.Errorf("failure running port forward process: %v", err)
+		message := "\n\nNotification: the cluster is IPv6 only cluster and it will break port-forward " +
```

Comment:
'the cluster is IPv6 only cluster' - this is probably not true for 99.9% of users seeing this error...

Reply:
Hi @howardjohn it might be confused, I change the message content, how about now?